src/subscription_manager/cli_command/environments.py

```diff
@@ -103,7 +103,7 @@ def _do_command(self):
             if self.options.set:
                 self._set_environments()
             else:
-                self._list_environments()
+                system_exit(os.EX_DATAERR, _("Error: At least one environment must be set."))
```

Comment:
This is not possible to done it this way, because if `subscription-manager environments --list` was used, then the `subscription-manager` would not returned requested list of environments, but it would returned this error.

You have to modify this code a little bit to something like this:

```python
elif self.options.list:
    self._list_environments()
else:
    system_exit(os.EX_DATAERR, _("Error: At least one environment must be set;"
                                 " use --list to get list of environments."))
```